dashboard/app/models/section.rb

```diff
@@ -129,6 +129,14 @@ def add_and_remove_student(student, old_section)
     add_student student
   end
 
+  # Figures out the default script for this section. If the section is assigned to
+  # a course rather than a script, it returns the first script in that course
+  # @return [Script]
+  def default_script
+    return script if script
+    return course.try(:course_scripts).try(:first).try(:script)
```

Comment:
Is this meant to return based on script position (in the course) or DB order (as implemented)?